tests/unit-test-run.mjs

```diff
@@ -0,0 +1,64 @@
+
+import serve from "./unit-test-server.mjs";
+import {
+  Builder,
+  By,
+  Capabilities,
+} from "selenium-webdriver";
+import assert from "assert";
+
+const PORT = 8010;
+const server = serve(PORT);
+
+let driver;
+
+async function test() {
+  let capabilities;
+  switch (process.env.BROWSER) {
+    case "safari":
+      capabilities = Capabilities.safari();
+      capabilities.set("safari.options", { technologyPreview: false });
+      break;
+
+    case "firefox": {
+      capabilities = Capabilities.firefox().setLoggingPrefs({ browser: "ALL" });
```

Comment:
Doesn't enabling extra logging  affect perf?

Reply:
Maybe, but this isn't actually running the benchmark or reporting results, this is only running the unit tests

Reply:
I'll just remove that from all of the browsers so it's consistent though, as I'm not seeing an observable change with or without it